project/Dependencies.scala

```diff
@@ -20,7 +20,7 @@ object Dependencies {
   val serviceTestV = s"0.19-$workbenchLibsHash"
   val workbenchModelV = s"0.14-$workbenchLibsHash"
   val workbenchGoogleV = s"0.21-$workbenchLibsHash"
-  val workbenchGoogle2V = s"0.21-$workbenchLibsHash"
+  val workbenchGoogle2V = "0.21-d89d5792-SNAP"
```

Comment:
we can probably use the snap version for now...and once we have the other wb-libs ugprade PR ready, we can then update the lib